Resources/en.lproj/Localizable.strings

```diff
@@ -51,10 +51,8 @@
 "CONTINUE_PLAYBACK" = "Continue playback?";
 "CONTINUE_PLAYBACK_LONG" = "Would you like to continue playback of \"%@\" where you left off?";
 
-"BUTTON_ALL" = "Select All";
```

Comment:
but Caro why the heck did you check in the Localizables .... yeah yeah I'm gonna remove that